src/main/fc/fc_rc.c

```diff
@@ -317,6 +324,16 @@ void updateRcCommands(void)
         rcCommand[ROLL] = rcCommand[ROLL] * cosDiff - rcCommand[PITCH] * sinDiff;
         rcCommand[PITCH] = rcCommand_PITCH;
     }
+
+    if (!ARMING_FLAG(ARMED)) {
+        if (IS_RC_MODE_ACTIVE(BOX3DDISABLESWITCH) && !reverseMotors) {
```

Comment:
There is work under way to extend the maximum number of possible modes. Once this is done, we'll probably want to assign this to a mode of its own. But for now this is good.

Also, what about moving this whole block up into `mwArm()`? That way, you make the code pretty robust against somebody accidentally changing it so that motors can be reversed for the mixer only, but not for blheli_s.